xdsl/dialects/memref.py

```diff
@@ -599,10 +599,6 @@ def get_canonicalization_patterns(cls) -> tuple[RewritePattern, ...]:
 class Subview(IRDLOperation):
 
     DYNAMIC_INDEX: ClassVar[int] = -9223372036854775808
-    """
-    Constant value used to denote dynamic indices in offsets, sizes, and strides.
-    Same constant as in MLIR.
-    """
```

Comment:
Was this intentional?

Reply:
casualty of reverting at the speed of sound 😔